### DISCUSSION TITLE
[BUG] 400 BadRequest Error  while registering with Pyarmor

### DISCUSSION THREAD

**Discussion by akhilmatta89** [dashingsoft/pyarmor#1079] [Q&A]
This issue is regarding the error which we have encountered while registering with PyArmor. It was consistently failing, most of the time with 400: Bad Request and sometimes with 503 Service Temporarily unavailable. Could you please guide us on fixing this?

Pyarmor version : - 6.7.3
Command used for registering :- pyarmor register pyarmor-regcode-1.txt

Please find the below screenshot on the same.

![image](https://user-images.githubusercontent.com/67899429/229427214-49ae6f2c-c313-4e3c-acaf-95e12685dea5.png)

**Comment by Carton32** [comment]
I have a related issue. In the logs, I can see:

```
| INFO     Getting remote file: https://pyarmor.dashingsoft.com/files/<xxxxxxxx>
| ERROR    <urlopen error timed out>
```

or 

`| ERROR    <urlopen error _ssl.c:703: The handshake operation timed out>`

The server is simply down or unresponsive.

**Comment by Carton32** [comment]
For me, it's working flawlessly again.

**Comment by jondy** [comment]
Please check 2 pinned issues

**Reply by akhilmatta89** [reply]
I didnt find any pinned comments links, can you please share me the links, so that i can go through them

**Reply by neetunarayanan** [reply]
@jondy we tried downloading the zip file; however, it gives error "This code has been used too many times". Could you please help us resolve this issue as we have an important release for which we need this issue to be resolved as soon as possible. Thank You.

**Comment by jondy** [comment]
https://github.com/dashingsoft/pyarmor/issues/980
https://github.com/dashingsoft/pyarmor/issues/985
